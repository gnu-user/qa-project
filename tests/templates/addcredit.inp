login
buyer
addcredit
10.00