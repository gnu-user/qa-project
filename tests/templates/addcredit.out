Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the amount of credit to add: 
Credit added successfully.